deno/src/main.ts

```diff
@@ -0,0 +1,208 @@
+// LICENSE = MIT
+import {
+  base64ToHex,
+  sha256,
+  shortHash,
+  shouldHash,
+  splitOnce,
+  urlSegments,
+} from "./utils.ts";
+
+interface Pkg {
+  module: string;
+  version: string;
+  name: string;
+  cpu?: "x86_64" | "aarch64";
+}
+
+export async function jsrPkgToFlatpakData(pkg: Pkg) {
+  const flatpkData = [];
+  const metaUrl = `https://jsr.io/${pkg.module}/meta.json`;
```

Comment:
is there a link to what this meta.json is?

Reply:
its just a meta file https://jsr.io/@std/io/meta.json, docs are here https://jsr.io/docs/api#jsr-registry-api

Reply:
It's required by Deno to analyze the deps and run the program